README.md

```diff
@@ -90,53 +90,23 @@ import com.dieam.reactnativepushnotification.ReactNativePushNotificationPackage;
 
 public class MainApplication extends Application implements ReactApplication {
 
-  private ReactNativePushNotificationPackage mReactNativePushNotificationPackage; // <------ Add Package Variable
```

Comment:
Since there are no more irregular steps to set up the package, maybe we should mention [rnpm](https://github.com/rnpm/rnpm) instead of having a step-by-step guide?